spec/unit/puppet/provider/ini_setting/ruby_spec.rb

```diff
@@ -840,13 +840,16 @@ def self.file_path
 foo= foovalue2
 baz=bazvalue
 url = http://192.168.1.1:8080
+[section3]
+# com = ment
+uncom = ment
+
+ -nonstandard-
+   shoes = purple
```

Comment:
I moved these lines up to verify that the `[section:sub]` is not affected by removing a section.